src/portable/Transcribe.Linux/packages.config

```diff
@@ -5,6 +5,7 @@
   <package id="icu.net" version="2.3.4" targetFramework="net461" />
   <package id="Microsoft.DotNet.PlatformAbstractions" version="2.0.4" targetFramework="net461" />
   <package id="Microsoft.Extensions.DependencyModel" version="2.0.4" targetFramework="net461" />
+  <package id="Microsoft-WindowsAPICodePack-Core" version="1.1.3.3" targetFramework="net461" />
```

Comment:
Yes, we should test to make sure this will do something helpful on Linux. Also I am not sure you are currently calling it from this program. Another way to handle it if it doesn't work on Linux and you still want to use it is by using dependency injection. The idea is you create a method in this Transcribe.Linux that works on Linux. You create a method with the same name and parameters in Transcribe.Windows that uses the NuGet package you would like to user. Then in you pass this method into ReactShared so it can be called when needed. This is the technique we are using for the avatar since it uses methods that are OS dependent and are not available in dot net core.